library/src/test/java/com/chuckerteam/chucker/ChuckerInterceptorDelegate.kt

```diff
@@ -12,10 +12,10 @@ import okhttp3.Response
 import java.util.concurrent.CopyOnWriteArrayList
 import java.util.concurrent.atomic.AtomicLong
 
-internal class ChuckerInterceptorDelegate(
-    fileFactory: FileFactory,
-    maxContentLength: Long = 250000L,
-    headersToRedact: Set<String> = emptySet()
+internal data class ChuckerInterceptorDelegate(
```

Comment:
Not sure I understood what this change gives us. Could explain, please?

Reply:
It comes from this test.

https://github.com/ChuckerTeam/chucker/blob/e3e259a46d057d06ca33387236a93e0ae7221d25/library/src/test/java/com/chuckerteam/chucker/api/ChuckerInterceptorTest.kt#L269-L282

Since the interceptor is created in `setUp` there is no way of changing its input parameters. I wanted to have more granular control over that and an explicit test.

Reply:
Now that I'm thinking `copy` is not needed. I could just create a new interceptor. What do you think?

Reply:
It might be a question of preference. I think creation of a new interceptor would be sort of more understandable for a new reader of the code.